test/test.ts

```diff
@@ -64,6 +64,46 @@ describe('DaiHrd', () => {
 		expect(await alice.pot.dsr_()).toEqual(newDsr)
 	})
 
+	it('can set DSR, deposit, and withdraw', async () => {
+		const DSR_RATE = 1.000001
+
+		const attodaiToDeposit = daiToAttodai(10_000n)
+		await generateDai(alice, attodaiToDeposit)
+
+		await mineBlock(alice)
+		expect(await alice.pot.dsr_()).toEqual(10n**27n)
+		const newDsr = xToRontox(DSR_RATE)
+		await alice.setDsr.setDsr(newDsr)
+		expect(await alice.pot.dsr_()).toEqual(newDsr)
+
+		await alice.dai.approve(alice.daiHrd.address, MAX_APPROVAL)
+		const chiBeforeDeposit = await alice.daiHrd.calculatedChi_()
+		await ganache.advanceTime(2)
+
+		await alice.daiHrd.deposit(attodaiToDeposit)
+		await ganache.advanceTime(2)
+
+		await mineBlock(alice)
+		const chiAfterDeposit = await alice.daiHrd.calculatedChi_()
+		await ganache.advanceTime(14)
```

Comment:
If we are using `ganache.advanceTime(...)`, it means that this test won't work in Parity.  This is fine, but in that case why not use `ganache.mine`?  It feels odd to do `ganache.advanceTime(...)` followed by a general purpose `mine`?

Reply:
well, advancing time and triggering a transaction are two different things, right?

Reply:
The one two lines up does:
```typescript
await ganache.advanceTime(2)
await mineBlock(alice)
```
Why not just do this?
```typescript
await ganache.mine(now + 2)
```

I suppose there is value in consistency and using separate methods for changing time and mining.